server/apps/main/models.py

```diff
@@ -5,6 +5,7 @@
 class PublicExperience(models.Model):
     experience_text = models.TextField()
     difference_text = models.TextField(default="")
+    title_text = models.TextField(default="")
```

Comment:
This is maybe a broader design question, but I wonder whether this field should be required rather than set to an empty default? As the *My Stories* and other UI parts focus heavily on the presence of a title it might be better to allow for that? @GeorgiaHCA do you know if a decision was made on that at some point?

Reply:
@gedankenstuecke it is actually required on the forms side in `share_experience.html`, which is AFAIK the only method of creating a story. So even though it is not required on the model view I think that users cannot create experiences without titles (or, currently, experiences or suggestions, see comment in PR issue). 

If you are dealing with the database locally you can still create entries without a title though.

Reply:
I have moved this to a discussion #400, because it is not currently actionable (since that is the criteria we discussed :) ).

Reply:
Yep, that makes sense!